manual/cops_sorbet.md

```diff
@@ -315,7 +347,7 @@ module SomeModule
 end
 
 # good
-module SomeModule;end
+module SomeModule; end
 ```
```

Comment:
Hmm, were we not keeping the docs in sync with the source? I guess this was not properly updated in the last PR, correct?

Reply:
My other PR that I merged should have this updated too tbh.